### ISSUE TITLE
iOS tapbar covering content at the bottom 

### ISSUE THREAD

**Issue by mxilievski** [UI-Lovelace-Minimalist/UI#1260]
**Describe the bug**
After updating to latest version of HA and UI minimalist, I’m not able to scroll to the bottom of the page to see the last bits of my components since they’re being covered with the tapbar. The theme I’m using is “minimalist-ios-tapbar”.

**Screenshots**
![Screenshot 2023-04-08 at 22 57 56](https://user-images.githubusercontent.com/2308950/230742304-328f0648-66db-460c-849f-1f9ac603194d.png)
![Screenshot 2023-04-08 at 23 00 16](https://user-images.githubusercontent.com/2308950/230742371-b3861fa6-d1e6-43c4-92b2-a8cd574bfcf1.png)

**Comment by mxilievski** [comment]
@basbruss I think I've found the issue. With [this](https://github.com/UI-Lovelace-Minimalist/UI/commit/92b6ad5d477781b0a6a398ca5bfeece0e09b9f72) commit you added this piece of code 
<img width="265" alt="image" src="https://user-images.githubusercontent.com/2308950/231114098-c0c496b3-21f2-4877-9883-11d9ffdffd0d.png">
However this is not overriding styles that are defined here 
<img width="446" alt="image" src="https://user-images.githubusercontent.com/2308950/231114289-0e942c27-1507-4b47-8057-b2b0fc5af108.png">

Basically with the code in `div#view` you're overriding the code in `#view` or in our case you're overriding both `
        margin-top: calc(-1 * var(--header-height)) !important;
        padding-bottom: var(--header-height) !important;
` 
which are needed.

**Comment by mxilievski** [comment]
@basbruss check the [PR](https://github.com/UI-Lovelace-Minimalist/UI/pull/1273).

**Comment by guger** [comment]
This bug happens again for my setup on both, ios-tapbar and mobile-tapbar. Any ideas how to fix this?

**Comment by mxilievski** [comment]
> This bug happens again for my setup on both, ios-tapbar and mobile-tapbar. Any ideas how to fix this?

fix waiting to be merged in https://github.com/UI-Lovelace-Minimalist/UI/pull/1604
